internal/driver/driver.go

```
@@ -73,7 +74,7 @@ func (d *Driver) HandleReadCommands(deviceName string, protocols map[string]cont
 				return responses, err
 			}
 
-			cv := sdkModel.NewStringValue(reqs[i].DeviceResourceName, 0, string(data))
+			cv, _ := sdkModel.NewCommandValue(reqs[i].DeviceResourceName, v2.ValueTypeString, string(data))
```

Should not ignore the error here and below.

refactor to add error handling after switch and move `responses[i] = cv` to after switch so it isn't repeated in every case.